src/main/java/frc/robot/utils/DeadbandProximity.java

```diff
@@ -0,0 +1,82 @@
+package frc.robot.utils;
+
+import edu.wpi.first.wpilibj.AnalogInput;
+
+/**
+ * This class is a modification of the regular proximity class, which adds a deadband, to insure that the sensor
+ * does not toggle multiple times.
+ * this is crucial for the conveyor, because the amount of balls that enter are counted by the amount of times
+ * the sensor switches.
+ */
+public class DeadbandProximity extends AnalogInput {
+    private final double lostVoltage;
+    private final double senseVoltage;
```

Comment:
"lost" is unclear. I think using "seen" and "unseen" are better and it's also clearer the two values are related.

Reply:
I disagree. In a technical view, the sensor can't "see", he doesn't have eyes, he only can sense.

Reply:
First of all, if a sensor can't sense, it's really misleading.
Secondly, why lost doesn't fit? it's pretty clear that it refers to the minimal voltage that the sensor sense, i.e. when the object is lost

Reply:
It fits, but "seen"/"unseen" or "sensed"/"unsensed" fit better, because it's clear the two flags are related to each other.
> First of all, if a sensor can't sense, it's really misleading.

If a sensor can't see because it doesn't have eyes, how can it sense if it doesn't have senses? The technical terms don't matter, only the terms we as humans use matter.

Reply:
I can use `ignoranceVoltage`, I don't agree that those words, they don't "tell" what is the real purpose of the variables

Reply:
They _do_ tell the real purpose: the outputted voltage when the sensor sees a Power Cell, and the outputted voltage when it doesn't see one.

Reply:
the sensor **can't** see, I'm not going to lie in my code, like in the shooter you fought with me to use `estimateVelocityFromDistance` just because it's more "correct", I'm not going to allow to change the name of the `lostVoltage` to another name as long as the name isn't more correct from my variable's name

Reply:
> you fought with me

I don't fight. I discuss. 🌷
> I'm not going to lie in my code

Not using the technical term ≠ lying.

Just gonna put here a few articles which also say that sensors see (emphasis mine):
- "Radar sensor for the home which can '_see_' through walls" ([source](https://www.bbc.com/news/av/technology-38608135))
- "it’s important to make sure each sensor has a clear _line of sight_" (sight ≡ see, [source](https://www.safewise.com/blog/effective-placement-can-prevent-a-burglary-where-your-sensors-should-go/))
- "What a Sensor _Sees_" ([source](https://www.te.com/global-en/trends/electric-vehicles-connected-transportation/what-a-sensor-sees-in-automotive.html))

This is my last comment on this matter unless you'd want to further discuss it.